examples/gno.land/p/moul/txlink/txlink.gno

```diff
@@ -5,13 +5,27 @@
 // flexible arguments, allowing users to build dynamic links that integrate
 // seamlessly with various Gno clients.
 //
-// The primary function, Call, is designed to produce markdown links for
-// transaction functions in the current "relative realm". By specifying a custom
-// Realm, you can generate links that either use the current realm path or a
-// fully qualified path for another realm.
+// The package offers a way to generate clickable transaction MD links
+// for the current "relative realm":
+//
+//  Using a builder pattern for more structured URLs:
+//     txlink.NewLink("MyFunc").
+//         AddArg("key", "value").      // add arguments one by one
+//         AddArgs("k1", "v1", "k2", "v2"). // or multiple at once
+//         SetSend("1000000ugnot").
+//         URL()
```

Comment:
Tell me you mostly write JS without telling me you mostly write JS ;)

(jk, it's fine, I like it)